database/scrapers/faq.ts

```diff
@@ -167,22 +99,12 @@ export default class FaqSectionScrapper extends BaseScraperModule {
           updatedAt = questionUpdatedAt;
         }
       }
-
-      let imageKey = "";
-      try {
-        imageKey = await this.uploadImage(article.cover);
-      } catch (error) {
-        this.logger.error(
-          `Failed to upload article cover image for article: ${article.id}: ${error}`,
-        );
-      }
-
       await GuideArticle.create({
         id: article.id,
         title: article.name,
         shortDesc: article.short_description,
         description: article.description ?? "",
-        imageKey,
+        imageKey: await this.directusUploadFieldAndGetKey(article.cover),
```

Comment:
since this is being called in a loop, consider using `.addErrorContext` to include a new error message which mentions which article this image belonged to